src/order/urls.py

```diff
@@ -7,11 +7,15 @@
     CreateOrder,
     UpdateOrder,
     UpdateOrderStatus,
-    DeleteOrder)
+    DeleteOrder,
+    placeMultipleOrders)
 urlpatterns = [
     url(_(r'^list/$'), OrderList.as_view(), name='list'),
     url(_(r'^view/(?P<pk>\d+)/$'), OrderDetail.as_view(), name='view'),
     url(_(r'^create/$'), CreateOrder.as_view(), name='create'),
+    # Multiple orders as once
+    url(_(r'^create/multiple/(?P<pk>\d+)$'),
```

Comment:
- Rather use: `create/batch/(?P<pk>\d+)$')`

- Use _createBatch_ rather than _placeMultipleOrders_ . Idem for the route name.